test/parallel/test-tls-securepair-fiftharg.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var common = require('../common');
+var assert = require('assert');
+var fs = require('fs');
+var tls = require('tls');
+
+var sslcontext = tls.createSecureContext({
+    cert: fs.readFileSync(common.fixturesDir + '/test_cert.pem'),
+    key: fs.readFileSync(common.fixturesDir + '/test_key.pem'),
+});
+var catched_servername = null;
+var pair = tls.createSecurePair(sslcontext, true, false, false, {
+    SNICallback: function(servername, cb) {
```

Comment:
In fact, this test can be written better with `common.mustCall`, like this
```js
SNICallback: common.mustCall(function(servername) {
  assert.strictEqual(servername, 'www.google.com');
})
```

You don't need the `catched_servername` variable.